website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx

```diff
@@ -0,0 +1,200 @@
+author: hermanschaaf
+---
+
+import { HowToGuideHeader } from "../../components/HowToGuideHeader"
+import { Callout } from 'nextra-theme-docs';
+
+<HowToGuideHeader/>
+
+MotherDuck is a serverless analytics service powered by DuckDB. If you have data in a Postgres database that you'd like to analyze using MotherDuck, this tutorial will show you how to do this using CloudQuery, an open source ELT framework. We will cover:
+ 
+ 1. How to copy a Postgres database into MotherDuck as one-off batch operation, and
+ 2. How to continuously stream Postgres data from Postgres to MotherDuck using Postgres Change Data Capture (CDC).
+
+Which option you choose is up to you. Either way, by the end of this tutorial you will be able to analyze data from your Postgres database using MotherDuck.
+
+## Step 0. Create a Test Table in Postgres (Optional)
+
+To demonstrate the steps in this tutorial, we will create a basic `customers` table in Postgres and insert 10,000 generated rows:
+
+```sql copy
+select
+  md5(random()::text) as id,
+  'person' || num || '@' ||
+  (case (random() * 2)::integer
+    when 0 then 'gmail'
+    when 1 then 'hotmail'
+    when 2 then 'yahoo'
+  end) || '.com' as email,
+  now() as created_at
+into customers
+from generate_series(1,10000) as num;
+alter table customers add primary key (id);
+```
+
+And let's test it out:
+
+```sql copy
+select * from customers limit 3;
+```
+
+```text
++----------------------------------+---------------------+------------------------------+
+| id                               | email               | created_at                   |
+|----------------------------------+---------------------+------------------------------|
+| 216016a4c29d66dd007cd1c9fa9994b4 | person1@hotmail.com | 2023-06-05 14:04:38.15548+01 |
+| de75a87f5f797248fd47ffcb15b7fff1 | person2@yahoo.com   | 2023-06-05 14:04:38.15548+01 |
+| 279838446e9890b00e92221f25bb0c73 | person3@gmail.com   | 2023-06-05 14:04:38.15548+01 |
++----------------------------------+---------------------+------------------------------+
+SELECT 3
+```
+
+## Step 1. Install CloudQuery
+
+CloudQuery is a cross-platform command-line tool for extracting and loading data that can be run just about anywhere: locally, on a virtual machine, or in a containerized environment. In this tutorial we'll try it out locally, and we'll use it to copy data from a local Postgres database to MotherDuck. On MacOS, you can install it using Homebrew:
+
+```shell copy
+brew install cloudquery/tap/cloudquery
+```
+
+See the [CloudQuery Quickstart guide](/docs/quickstart) for installation instructions on other platforms. Please note that the DuckDB plugin (that we will use later) [does not support Windows](https://github.com/cloudquery/cloudquery/issues/8854) at this time.
+
+Once CloudQuery is installed, you should be able to invoke it from the command line:
+
+```shell
+$ cloudquery --version
+cloudquery version 3.5.0
```

Comment:
Latest CLI version from Brew is 3.5.4
```suggestion
cloudquery version 3.5.4
```

Maybe the version placeholder will work too